test/scripts/bulk-mailer.js

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const ROOT_DIR = '../..'
+
+const { assert } = require('chai')
+const cp = require('child_process')
+const fs = require('fs')
+const mkdirp = require('mkdirp')
+const mocks = require(`${ROOT_DIR}/test/mocks`)
+const path = require('path')
+const P = require('bluebird')
+const rimraf = require('rimraf')
+
+const cwd = path.resolve(__dirname, ROOT_DIR)
+cp.execAsync = P.promisify(cp.exec)
+
+const log = mocks.mockLog()
+const config = require('../../config').getProperties()
+const Token = require('../../lib/tokens')(log, config)
+const UnblockCode = require('../../lib/crypto/random').base32(config.signinUnblock.codeLength)
+
+const OUTPUT_DIRECTORY = path.resolve(__dirname, './test_output')
+const USER_DUMP_PATH = path.join(OUTPUT_DIRECTORY, 'user_dump.json')
+
+describe('scripts/bulk-mailer', () => {
+  let db;
+
+  before(() => {
+    const DB = require('../../lib/db')(
+      config,
+      log,
+      Token,
+      UnblockCode
+    )
+
+    rimraf.sync(OUTPUT_DIRECTORY)
+    mkdirp.sync(OUTPUT_DIRECTORY)
+
+    return DB.connect(config[config.db.backend])
+      .then(_db => {
+        db = _db;
+        return db.createAccount({
```

Comment:
Might make testing a little cleaner if you mock account object

```javascript
const accountMock = {
          email: 'user2@test.com',
          uid: 'f9916686c226415abd06ae550f073ced',
}
```

so you can reference `accountMock.uid` instead of the uid hex value.